client/src/CommentBox.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { PropTypes } from 'react';
+import $ from 'jquery';
+import CommentForm from './AddCommentForm';
+import CommentList from './CommentList';
+
+class CommentBox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { data: [] };
+  }
+
+  componentDidMount = () => {
+    this.loadComments();
+    setInterval(this.loadComments, this.props.pollInterval);
+  }
+
+  loadComments = () => {
+    $.ajax({
+      url: this.props.url.concat('/', this.props.poemId),
+      dataType: 'json',
+      cache: false,
+      success: (data) => {
+        this.setState({ data });
+      },
+      error: (xhr, status, err) => {
+        console.error(this.props.url, status, err.toString());
+      },
+    });
+  }
+
+  handleCommentSubmit = (comment) => {
+    const comments = this.state.data;
+    const mutatedComment = comment;
+    mutatedComment.poemId = this.props.poemId;
+    mutatedComment.id = Date.now();
+    const newComments = comments.concat([mutatedComment]);
+    this.setState({ data: newComments });
+    $.ajax({
+      url: this.props.url,
+      dataType: 'json',
+      type: 'POST',
+      data: comment,
```

Comment:
We'll need to `JSON.stringify` the data, as well as set the content type of the request to `application/json; charset=utf-8`.